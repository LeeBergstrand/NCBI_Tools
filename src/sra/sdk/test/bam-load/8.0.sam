@CO	Hard clipping
@CO	-----------------------------------------------------
@HD	VN:1.0	GO:none	SO:coordinate
@SQ	SN:20	LN:63025520	AS:NCBI37	
SRR006423.6228848	98	20	60025	60	35M16H	=	60079	89	AGCTTGGAACCCTATAGAGTTGCTGAGTGCCAGGA	=;>@?;%=<>@AC@B79,>2C4B;7877%7%58:6	XC:i:35	XT:A:U	NM:i:0	SM:i:37	AM:i:37	X0:i:1	X1:i:0	XM:i:0	XO:i:0	XG:i:0	MD:Z:35	RG:Z:SRR006423	OQ:Z:IIIIII%I<EFIGD>-4&I(B*@++++)%)%'+0(
