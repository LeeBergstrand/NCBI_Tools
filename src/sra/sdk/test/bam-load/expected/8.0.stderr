2012-11-26T22:11:14 bam-load.2.2.1 err: libs/align/writer-reference.c:cigar2offset:2162: data invalid while processing file within alignment module - Hard clipping of sequence data is not allowed
2012-11-26T22:11:14 bam-load.2.2.1 warn: libs/align/writer-reference.c:cigar2offset:2162: data invalid while processing file within alignment module - Spot 'SRR006423.6228848': bad alignment to reference '20' at 60024
2012-11-26T22:11:14 bam-load.2.2.1 err: libs/kfs/md5.c:KMD5SumFmtDelete:680: data invalid while processing file within alignment module - accession="8.0-obj" errors="1" status="failure"
2012-11-26T22:11:14 bam-load.2.2.1 err: libs/kfs/md5.c:KMD5SumFmtDelete:680: data invalid while processing file within alignment module - load failed
